product_sticker/models/product_sticker.py

```diff
@@ -0,0 +1,208 @@
+from odoo import api, fields, models, tools
+from odoo.osv import expression
+from odoo.tools import html2plaintext
+
+
+class ProductSticker(models.Model):
+    _name = "product.sticker"
+    _description = "Product Sticker"
+    _inherit = ["image.mixin"]
+    _order = "sequence, id"
+
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        string="Company",
+        default=lambda s: s.env.company,
+    )
+    sequence = fields.Integer(default=10, index=True)
+    name = fields.Char(required=True, translate=True)
+    image_1920 = fields.Image(required=True)
+    image_1920_size = fields.Integer(
+        compute="_compute_image_1920_size",
+        store=False,
+    )
+    product_category_id = fields.Many2one(
+        comodel_name="product.category",
+        string="Category",
+        ondelete="cascade",
+    )
+    product_attribute_id = fields.Many2one(
+        comodel_name="product.attribute",
+        string="Attribute",
+        ondelete="cascade",
+    )
+    product_attribute_value_id = fields.Many2one(
+        comodel_name="product.attribute.value",
+        string="Attribute value",
+        ondelete="cascade",
+        domain="[('attribute_id', '=', product_attribute_id)]",
+    )
+    can_image_128_be_zoomed = fields.Boolean(
+        string="Allow zooming in 128px image",
+        compute="_compute_can_image_128_be_zoomed",
+        store=True,
+    )
+    show_sticker_note = fields.Boolean(
+        string="Sticker Note",
+        help="If checked, the note will be displayed with the sticker",
+    )
+    note = fields.Text(
```

Comment:
I don't think it's a good idea to make users write HTML. Let's just use the appropriate field, don't you think?

```suggestion
    note = fields.Html(
```

Reply:
No, it's better to use a simple text label and display always the same format. It's like a caption image.